.github/workflows/snyk-python-scan.yml

```diff
@@ -0,0 +1,54 @@
+name: 🚨 Snyk Python Scan
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 3'
```

Comment:
@pmaslana, should this follow the standard pattern (plus cron)?

https://github.com/Chia-Network/chia-blockchain/blob/939b129f9f72dc3ee2aea0498c22363fac16e141/.github/workflows/test.yml#L3-L13